packages/cli/src/lib/webpack/render-html-plugin.js

```diff
@@ -78,29 +98,44 @@ module.exports = async function renderHTMLPlugin(config, env) {
 		return {
 			title,
 			filename,
-			template: `!!${require.resolve('ejs-loader')}?esModule=false!${template}`,
+			template: `!!${require.resolve(
+				'ejs-loader'
+			)}?esModule=false!${templatePath}`,
 			templateParameters: async (compilation, assets, assetTags, options) => {
 				let entrypoints = {};
 				compilation.entrypoints.forEach((entrypoint, name) => {
 					let entryFiles = entrypoint.getFiles();
+
 					entrypoints[name] =
 						assets.publicPath +
-						entryFiles.find(file => /\.(m?js)(\?|$)/.test(file));
+						entryFiles.find(file => /\.m?js(?:\?|$)/.test(file));
 				});
 
+				const compilationAssets = Object.keys(compilation.assets);
+				const outputName = entrypoints['bundle'].match(/^([^.]*)/)[1];
+				const rgx = new RegExp(`${outputName}.*legacy.js$`);
+				const legacyOutput = compilationAssets.find(file => rgx.test(file));
+				if (legacyOutput) {
+					entrypoints['legacy-bundle'] = assets.publicPath + legacyOutput;
+				}
+
+				if (compilationAssets.includes('es-polyfills.js')) {
+					entrypoints['es-polyfills'] = assets.publicPath + 'es-polyfills.js';
+				}
+
 				return {
 					cli: {
 						title,
 						url,
 						manifest: config.manifest,
-						inlineCss: config['inlineCss'],
```

Comment:
Already available as `config.inlineCss`, no point in having a duplicated top level key for it.